API/models/model.go

```diff
@@ -124,6 +124,29 @@ func PropagateParentIdChange(ctx context.Context, oldParentId, newId string, ent
 	return nil
 }
 
+// PropagateDomainChange: search for all objects with reference to the modified domain
+func PropagateDomainChange(ctx context.Context, oldDomainId, newDomainId string) error {
+	// Find all objects containing this domain
+	req := bson.M{"domain": primitive.Regex{Pattern: "^" + oldDomainId + "(\\" + u.HN_DELIMETER + "|$)", Options: ""}}
+	// For each object found, replace old domain by new
+	update := bson.D{{
+		Key: "$set", Value: bson.M{
+			"domain": bson.M{
+				"$replaceOne": bson.M{
+					"input":       "$domain",
+					"find":        oldDomainId,
+					"replacement": newDomainId}}}}}
+	for i := u.STRAYOBJ + 1; i <= u.GROUP; i++ {
+		_, e := repository.GetDB().Collection(u.EntityToString(i)).UpdateMany(ctx,
```

Comment:
Normally, in golang the variables that are of type error are called err (or at least that's what I always see in big libraries/stackoverflow).

Reply:
fixed it